server/tests/src/main/scala/sttp/tapir/server/tests/ServerContentNegotiationTests.scala

```diff
@@ -79,6 +79,23 @@ class ServerContentNegotiationTests[F[_], OPTIONS, ROUTE](createServerTest: Crea
       testNameSuffix = "not take into account the accept charset header when the body media type doesn't specify one"
     )(in => pureResult(in.asRight[Unit])) { (backend, baseUri) =>
       basicRequest.post(uri"$baseUri/api/echo").header(HeaderNames.AcceptCharset, "utf8").send(backend).map(_.code shouldBe StatusCode.Ok)
+    },
+    testServer(out_json_json_different_parameters, testNameSuffix = "matches content type on accept parameters")(_ =>
+      pureResult(Organization("sml").asRight[Unit])
+    ) { (backend, baseUri) =>
+      val r1 = basicRequest
+        .header(HeaderNames.Accept, "application/json; name=unknown")
+        .get(uri"$baseUri/content-negotiation/organization-parameters")
+        .send(backend)
+        .map(_.body shouldBe Right("{\"name\":\"unknown\"}"))
+
+      val r2 = basicRequest
+        .header(HeaderNames.Accept, "application/json")
```

Comment:
@kamilkloch I also had to update this test, otherwise the assumed `Accept` header was `*/*`, which also corresponded to the first variant

Reply:
Thank you, I did the same, you were faster :) I also added the 3rd variant in the test, to catch the '*/*' variant.